### DISCUSSION TITLE
Discord rsxp-2023 Channel

### DISCUSSION THREAD

**Discussion by magnokf** [diego3g/rsxp-2023#30] [General]
Seria uma ótima ideia existir um canal voltado somente para esse fim no Discord da RocketSeat.
O que acham?

Já exite?

**Comment by diego3g** [comment]
Você diz um canal pra falarmos sobre o desenvolvimento do app?

O que me preocupa só é ser um canal aberto e vir gente de todo lado falando coisas sem contexto ou se for um canal fechado como que daríamos acesso às pessoas.

**Reply by magnokf** [reply]
Sim, um canal exclusivo para quem realmente está comprometido em ajudar, colaborar e aprender. Tipo um canal para seus alunos do Ignite por exemplo.

**Comment by LuisCarlosDev** [comment]
Canal fechado acho que daria certo!

**Comment by LuizHAP** [comment]
Acredito que podemos manter por aqui a maioria das discussões. Não sei se faz sentido mais um ponto de comunicação

**Reply by carlosdavid0** [reply]
Porém, acho que faz sentido para uma comunicação mais rápida, ainda formalizando tudo por aqui.

**Comment by diego3g** [comment]
Galera, prefiro manter por aqui pra não dividir as discussões. Se criarmos um Discord, logo vamos ter coisas sendo discutidas lá dentro que não virão para dentro do Github e perdemos o histórico de discussão e documentação do projeto.
